### ISSUE TITLE
Picker tests failing

### ISSUE THREAD

**Issue by phatmann** [kif-framework/KIF#590]
-[PickerTests testSelectingDateTime] failed: The step timed out after 10.00 seconds: Found an accessibility element with the label "Date Time Selection", but with the value "Monday, Jun 17, 06:43 AM", not "Sunday, Jun 17, 06:43 AM"
